include/ceed.h

```diff
@@ -349,6 +349,8 @@ CEED_EXTERN int CeedOperatorSetField(CeedOperator op, const char *fieldname,
                                      CeedVector v);
 CEED_EXTERN int CeedCompositeOperatorAddSub(CeedOperator compositeop,
     CeedOperator subop);
+CEED_EXTERN int CeedOperatorLinearizeQFunction(CeedOperator op,
```

Comment:
I could add an interface to handle qfunctions that are already linear. I wanted to start with something more general.